packages/tools/client-debugger/client-debugger-view/src/components/ContainerView.tsx

```diff
@@ -2,16 +2,56 @@
  * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
  * Licensed under the MIT License.
  */
-import { IOverflowSetItemProps, IconButton, Link, OverflowSet, Stack } from "@fluentui/react";
+import { IOverflowSetItemProps, Stack } from "@fluentui/react";
+
+import {
+	makeStyles,
+	shorthands,
+	tokens,
+	Tab,
+	TabList,
+	SelectTabData,
+	SelectTabEvent,
+	TabValue,
+} from "@fluentui/react-components";
 import { HasContainerId } from "@fluid-tools/client-debugger";
-import React from "react";
+import * as React from "react";
```

Comment:
nit: not fully sure if this will pass the styling/prettier check?

Reply:
No issues with the lint check!